### PR TITLE
LOOP-1495: Introduce Therapy Settings into acceptance flow

### PR DESCRIPTION
**Pull request by rickpasetto**
This refactors a little to allow for an "action button" at the bottom.

[LOOP-1495](https://tidepool.atlassian.net/browse/LOOP-1495)

### CODE REVIEW

File: LoopKitUI/Views/Settings Editors/TherapySettingsView.swift
Comment (rickpasetto) [inline LoopKitUI/Views/Settings Editors/TherapySettingsView.swift:21]: Moved `mode` into `TherapySettingsViewModel` because the ViewModel is what is shared amongst the screens in the two "incarnations" of editing/reviewing TherapySettings.  Seemed to make sense to me.

**Review by mpangburn — APPROVED** [review]
LGTM!

**Review by novalegra — APPROVED** [review]
(see question below, I noticed it after I approved)

File: LoopKitUI/Views/Settings Editors/TherapySettingsView.swift
Comment (novalegra) [inline LoopKitUI/Views/Settings Editors/TherapySettingsView.swift:48]: [design question] When I built the overview screen, I noticed there aren't lines between the individual settings - is that intentional? (the design currently includes them)

**Review by novalegra — COMMENTED** [review]
One design ?, otherwise LGTM

File: LoopKitUI/Views/Settings Editors/TherapySettingsView.swift
Comment (novalegra) [reply LoopKitUI/Views/Settings Editors/TherapySettingsView.swift:48]: Build
Screen Shot 2020-07-24 at 4 40 09 PM

Design
Screen Shot 2020-07-24 at 4 40 26 PM

Comment (rickpasetto) [reply LoopKitUI/Views/Settings Editors/TherapySettingsView.swift:48]: @novalegra I'm pretty sure I'm doing whatever the default is.  I can ask at standup today, but since this is unrelated to this PR, can I please get an approval?  I can make a followup PR if there's a need...

Comment (novalegra) [reply LoopKitUI/Views/Settings Editors/TherapySettingsView.swift:48]: Sure!

**Review by novalegra — APPROVED** [review]
Lingering style question, but LGTM! 👍

**Pull request merged**
